apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -152,9 +152,7 @@ const Ticket = new GQLListSchema('Ticket', {
             schemaDoc: 'Typification / classification / types of work',
             type: Relationship,
             ref: 'TicketClassifier',
-            isRequired: true,
-            knexOptions: { isNotNullable: true }, // Required relationship only!
-            kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
```

Comment:
Is it for future transition to new 3-level classifier?

Reply:
this is because we do not pass the classifier from a mobile device

Reply:
Could you write a comment about why it is not required, to make it easier to figure out the reason in the future.

Reply:
wrote a comment in 68e5bad